src/libs/ga-analytics.js

```diff
@@ -0,0 +1,17 @@
+import ReactGA from 'react-ga4';
+
+const TRACKING_ID = 'G-SDBSSSN65X';
+
+export function init() {
+  // Enable testMode on local or development environment
+  const isDev = !process.env.NODE_ENV || process.env.NODE_ENV === 'development';
+  ReactGA.initialize(TRACKING_ID, { testMode: isDev });
```

Comment:
Oooh, nice! TIL on this kind of usage: https://github.com/PriceRunner/react-ga4/issues/13#issuecomment-917444954.